ps_emailalerts.php

```diff
@@ -122,15 +122,18 @@ public function install($delete_params = true)
             Configuration::updateGlobalValue('MA_PRODUCT_COVERAGE', 0);
 
             $sql = 'CREATE TABLE IF NOT EXISTS `'._DB_PREFIX_.MailAlert::$definition['table'].'`
-				(
-					`id_customer` int(10) unsigned NOT NULL,
-					`customer_email` varchar(128) NOT NULL,
-					`id_product` int(10) unsigned NOT NULL,
-					`id_product_attribute` int(10) unsigned NOT NULL,
-					`id_shop` int(10) unsigned NOT NULL,
-					`id_lang` int(10) unsigned NOT NULL,
-					PRIMARY KEY  (`id_customer`,`customer_email`,`id_product`,`id_product_attribute`,`id_shop`)
-				) ENGINE='._MYSQL_ENGINE_.' DEFAULT CHARSET=utf8 COLLATE=utf8_general_ci';
+                (
+                    `id_mailalert` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
```

Comment:
```suggestion
                    `id_emailalert` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
```
To be consistent, you also need to change the primary key.
Add an upgrade script, and change the MailAlert definition

Reply:
Hi @PierreRambaud , i have changed as you suggested... PR 0c72a56 . Are ok now?